src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests"), 
+           InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
Interesting! This looks cool :smile: 

Does this just allow us to test internal classes from `Public.Data.Api` in `Public.Data.Api.Tests`?

Reply:
Yeah exactly! :)

It lets us keep things `internal` that should be `internal` but still allows us to test them in `Public.Data.Api.Tests`.

Usually 1 line is enough, but I had to add the `DynamicProxyGenAssembly2` reference too due to using the `Moq` framework in conjunction with an `internal` class I think (from memory). So this one was a 2 liner